Gemfile

```diff
@@ -9,5 +9,5 @@ gem 'workarea-testing', path: 'testing'
 gem 'listen'
 gem 'bundler-audit'
 gem 'rubocop'
-gem 'rails-decorators', git: 'https://github.com/workarea-commerce/rails-decorators.git'
+gem 'rails-decorators', git: 'https://github.com/workarea-commerce/rails-decorators.git', branch: 'master'
```

Comment:
I can remove this if you folks want me to, but this makes it a bit easier to work with master branch of workarea since I have a local override on my system for rails-decorators so it's a bit less cumbersome to work with experimental branches across multiple repos.

Reply:
I don't quite follow - aren't these two lines equivalent given `rails-decorators` default branch is `master`?

Reply:
@bencrouse sorry, never actually mentioned that I was receiving an error when you run `bundle` if you have a local override set up for the gem and it's pointing to a Git repo. Basically, since the Bundle is depending on `master` branch, the local override needs to also have `master` branch checked out in order to make sure the code is the same.